docs/tutorial/population_dynamics.rst

```diff
@@ -124,55 +173,82 @@ bins to be plotted as `n_bins`.
+    plt.figure()
+    dwell_2.hist(bin_spacing="log")
+    plt.show()
 
-Here we see that the double exponential fit visually looks better and the log likelihood is also higher than that
-for the single exponential fit. However, the log likelihood does not take into account model complexity, and will
-always increase for a model with more degrees of freedom. Instead, we can look at the Bayesian Information Criterion (BIC)
-or Akaike Information Criterion (AIC) to determine which model is better::
+.. image:: figures/population_dynamics/dwell2_hist.png
 
-    >>> print(dwell_1.bic, dwell_1.aic)
-    532.3299315589168  529.0366267341923
+Here we see visually that there is not significant improvement in the quality of the fit, so the single exponential is probably a better
+model for this data.
 
-    >>> print(dwell_2.bic, dwell_2.aic)
-    520.4562630650156  510.5763485908421
+We can also use some statistics to help choose the most appropriate model. The MLE method maximizes a :ref:`likelihood function <exponential_likelihood>`,
+with the final value reported in the legend of the histogram. We see that the likelihood of the double exponential model is slightly higher
+than that of the single exponential model which would indicate naively that the double exponential model is better. However,
+the likelihood does not take into account model complexity and will always increase with increasing number of adjustable parameters.
 
-These information criterion values weigh the log likelihood against the model complexity, and as such are more useful for
-model selection. In general, the model with the lowest value is optimal. We can see that both values are lower for the double
-exponential model, indicating that it is a better fit to the data.
+More informative statistics for model comparison are the Information Criteria. Two specific criteria are available from the model:
+the Bayesian Information Criterion (BIC) and the Akaike Information Criterion (AIC)::
 
-We can see this effect if we purposely overfit the data. The following plot shows the result of fitting simulated data (randomly sampled
-from a single exponential distribution) with either a one- or two-component model. In the figure legends we see that the log likelihood
-increases slightly for the two-component model because of the larger degrees of freedom. However, the BIC for the one-component model is
-indeed lower, as expected:
+    print(dwell_1.bic, dwell_1.aic)  # -7597.384625071581 -7602.4312723494295
+    print(dwell_2.bic, dwell_2.aic)  # -7602.027247179558 -7617.167189013104
 
-.. image:: kymo_bic_compare.png
+These information criterion values weigh the log likelihood against the model complexity, and as such are more useful for
+model selection. In general, the model with the lowest value is optimal. We can see that both values are lower for the double
+exponential model, but only slightly so it is not strong evidence to choose the more complex model.
```

Comment:
I don't want to put too much detail here at the moment since these values generally aren't the *best* way to choose the appropriate model but they are commonly used or talked about so I also don't want to skip over it. We have some plans for better uncertainty analysis and model comparisons planned so we can expand on those points when these features get implemented